website/docs/docs/partnerships.mdx

```diff
@@ -39,7 +39,7 @@ Our Workflow Partners typically have the following use cases:
 - **CI/CD:** These partners focus on continuous integration and continuous delivery/deployment.
 - **VCS:** These partners focus on tracking and managing software code changes.
 
-Most workflow partners integrate with the Terraform workflow itself. Run tasks allow Terraform Cloud to execute tasks in external systems at specific points in the Terraform Cloud run lifecycle. This offers much more extensibility to Terraform Cloud customers, enabling them to integrate your services into the Terraform Cloud workflow. The beta release of this feature allows users to add and execute these tasks during the new pre-apply stage which exists in between the plan and apply stages. Eventually, we will open the entire workflow to Terraform Cloud users, including the pre-plan and post apply stages. Reference the [Terraform Cloud Integrations](/docs/partnerships#terraform-cloud-integrations) or [Terraform Enterprise Integrations](/docs/partnerships#terraform-enterprise-integrations) documentation for more details. 
+Most workflow partners integrate with the Terraform workflow itself. Run tasks allow Terraform Cloud to execute tasks in external systems at specific points in the Terraform Cloud run lifecycle. This offers much more extensibility to Terraform Cloud customers, enabling them to integrate your services into the Terraform Cloud workflow. This feature allows users to add and execute these tasks during the post-plan and the new pre-plan (public beta) stages. Eventually, we will open the third phase of this workflow to Terraform Cloud users, the post apply stage. Reference the [Terraform Cloud Integrations](/docs/partnerships#terraform-cloud-integrations) or [Terraform Enterprise Integrations](/docs/partnerships#terraform-enterprise-integrations) documentation for more details. 
```

Comment:
A couple of thoughts on this. First, should we link to the actual run tasks page so folks can get more details about what run tasks are? Also I'm just thinking about how we can make things flow and make it more concise. Oh and finally --> we really try to stay away from future-facing promises in docs. Just because things change, and we've been burned by this before. Instead, i think we should just tell folks the specific points that TFC users can currently choose to execute run tasks.
Thoughts? 

```suggestion
Most workflow partners create [run tasks](/cloud-docs/integrations/run-tasks) that let them integrate their services directly into the Terraform workflow. [Run tasks](/cloud-docs/integrations/run-tasks) let Terraform Cloud execute tasks in external systems at certain points in the Terraform Cloud run lifecycle. Specifically, Terraform Cloud users can configure run tasks to execute during the post-plan and the pre-plan (public beta) run stages. Refer to the [Terraform Cloud Integrations](/docs/partnerships#terraform-cloud-integrations) or [Terraform Enterprise Integrations](/docs/partnerships#terraform-enterprise-integrations) documentation for more details. 
```

Reply:
Yup, sounds good!